Firebend.AutoCrud.ChangeTracking.EntityFramework/Implementations/EntityFrameworkChangeTrackingService.cs

```diff
@@ -61,7 +62,7 @@ public Task TrackUpdateAsync(EntityUpdatedDomainEvent<TEntity> domainEvent, Canc
             Source = domainEvent.EventContext?.Source,
             UserEmail = domainEvent.EventContext?.UserEmail,
             Action = action,
-            Changes = operations,
+            Changes = operations?.Select(x => new Operation(x.op, x.path, x.from, x.value)).ToList(),
```

Comment:
could we create an extension method for this? i know the `DefaultJsonPatchGenerator` can do it also. Just wanted to keep this DRY in case in future version they change the constructor for operations